cms/templates/js/active-video-upload-list.underscore

```diff
@@ -6,5 +6,5 @@
 </form>
 <section class="active-video-upload-container">
     <h3 class="sr"><%= gettext("Active Uploads") %></h3>
-    <ul class="active-video-upload-list"></ul>
+    <ul class="active-video-upload-list" aria-live="polite"></ul>
```

Comment:
I'm not hearing the update at all when I use Vocieover. Using NVDA in Windows, I hear "[filename] Uploading" twice.   Can you try removing aria-live and using ```role="alert"```  instead? Also, There is no indication that the file uploaded. Upon successful upload, can we add a message that says "Upload complete"?

I also had an issue with focus going back to the top of the page.  I would expect it to remain on the file upload button.  Is the page being reloaded?  If so, we may want to move focus back to the file upload button.